entries/d/dotdigital.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "dotdigital": {

```

Comment:
```suggestion
  "Dotdigital": {

```

Reply:
I understand if it is on grammar grounds but the name of the product is all lower case and written as dotdigital in materials. Do you still want me to capitalise it?